test/v2/controllers/validators/RetrieveUkPropertyAnnualSubmissionValidatorFactorySpec.scala

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.validators
+
+import api.models.domain.{BusinessId, Nino, TaxYear}
+import api.models.errors._
+import mocks.MockAppConfig
+import support.UnitSpec
+import v2.models.request.retrieveUkPropertyAnnualSubmission.RetrieveUkPropertyAnnualSubmissionRequestData
+
+class RetrieveUkPropertyAnnualSubmissionValidatorFactorySpec extends UnitSpec with MockAppConfig {
+
+  val validNino: String              = "AA123456B"
+  val validBusinessId: String        = "XAIS12345678901"
+  val validTaxYear: String           = "2021-22"
+  implicit val correlationId: String = "X-123"
```

Comment:
can make all these vals private and can remove the type of `String`

Reply:
didn't mean to include the correlationId, that needs to remain typed as a String as its an implicit